domains/rahul-krishnan.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rahulk98",
+        "email": "rahul.krishnan27@gmail.com"
+    },
+    "records": {
+        "CNAME": "https://rahulk98.github.io/glowing-spork/"
```

CNAME doesnt allow trailing slashes (/)
```suggestion
        "CNAME": "rahulk98.github.io"
```